### DISCUSSION TITLE
Trying to build on Ubuntu 22.04 ... "Unsupported or unknown distribution"

### DISCUSSION THREAD

**Discussion by rao94583** [AcademySoftwareFoundation/OpenRV#331] [Build Issues]
Seems like it ought to be build on 22.04, given that our Autodesk RV (for centos) does work fine on it.

I went through the whole set up rigmarole, and finally ended up with:
`ubuntu:~/Code/OpenRV$ cmake -B _build -DCMAKE_BUILD_TYPE=Release -DRV_DEPS_QT5_LOCATION=/usr/include/x86_64-linux-gnu/qt5
-- The CXX compiler identification is GNU 11.4.0
-- The C compiler identification is GNU 11.4.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Build type: Release
PROJECT DIR /home/ubuntu/Code/OpenRV
-- Building RV for generic Linux OS
CMake Error at cmake/defaults/rv_targets.cmake:116 (MESSAGE):
  Unknown or unsupported Linux distribution version; stopping configuration!
Call Stack (most recent call first):
  CMakeLists.txt:101 (INCLUDE)

-- Configuring incomplete, errors occurred!
See also "/home/ubuntu/Code/OpenRV/_build/CMakeFiles/CMakeOutput.log".`

**Comment by geffrak** [comment]
The built environment has been created and tested on RHEL. Feel free to submit a PR enabling the build on Ubuntu if you managed to have a complete build on that platform.

**Comment by jorxster** [comment]
We are also interested in building on Ubuntu and looking into what would be needed.
